.github/pull_request_template.md

```diff
@@ -1,16 +1,16 @@
 <!-- What issue does this PR close? -->
-Closes #
-
+Closes #3046 Adding the line break for better UI on Recent Changes page
```

Comment:
@yash-gupta2000 You have edited the pull_request_template.md. Please revert back the changes in this file